handlers/additem.js

```diff
@@ -152,12 +155,16 @@ const notifyAdditemSuccess = async function (query, itemName, order_id) {
     const replyListenerId = await bot.onReplyToMessage(query.message.chat.id, query.message.message_id, async (replymsg) => {
         try {
             if (await queries.hasOrder(replymsg.reply_to_message.message_id)) {
-                await queries.addRemark({
-                    remarks: replymsg.text,
-                    message_id: replymsg.reply_to_message.message_id,
-                });
-                messenger.send(replymsg.chat.id, 'Remark added successfully!');
-                queries.refreshLiveCountMessage(await queries.getChatIdFromOrderId(order_id))
+                if (replymsg.text.length > 60) {
```

Comment:
please abstract these values to a const in config.js

Reply:
Done!